src/components/vega-wallet/new-wallet-available.tsx

```diff
@@ -0,0 +1,56 @@
+import "./new-wallet-available.scss";
+
+import * as Sentry from "@sentry/react";
+import React from "react";
+import { useTranslation } from "react-i18next";
+
+import { Links } from "../../config/links";
+import { vegaWalletService } from "../../lib/vega-wallet/vega-wallet-service";
+
+export const NewWalletAvailable = () => {
+  const { t } = useTranslation();
+  const [newVersionAvailable, setNewVersionAvailable] = React.useState<
+    string | null
+  >(null);
+  const [latestVersionLocation, setLatestVersionLocation] =
```

Comment:
Minor nitpick: Can we make this consistent to the state above so it expects string | null and is initialized with null